llvm/test/CodeGen/SPIRV/hlsl-resources/UniqueImplicitBindingNumber.ll

```diff
@@ -0,0 +1,20 @@
+; RUN: llc -O0 -verify-machineinstrs -mtriple=spirv1.6-vulkan1.3-library %s -o - 
```

Comment:
If you expect this to have an error. You can do

```suggestion
; RUN: not llc -O0 -verify-machineinstrs -mtriple=spirv1.6-vulkan1.3-library %s -o - 
```

Then you can add `FileCheck` to look for the error message you expect.